docs/GettingStarted.md

```diff
@@ -0,0 +1,103 @@
+# Getting Started
+
+## Installing React Native and its dependencies
+
+This is a targeted summary of setup React Native setup for specifically React Native Windows. See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for full details.
+
+#### Dependencies: Node, Python, JDK
+
+Install the dependencies [specified by React Native](http://facebook.github.io/react-native/docs/getting-started.html#node-python2-jdk). Specifically, make sure a recent version of Node, Python2, and a recent version of the Java SE Development Kit (JDK) are installed.
```

Comment:
As far as I know, neither Python or the JDK are required for React Native Windows development. I think those are used for React Native Android development, which doesn't apply for this document.

Reply:
That is a good point! Do you agree @rozele @sayar ?

Reply:
Correct, those are only needed for Android.